.github/workflows/wheels.yml

```diff
@@ -38,15 +38,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        os: [ubuntu-20.04, windows-2019, macos-11]
 
     steps:
     - uses: actions/checkout@v3
 
     - uses: pypa/cibuildwheel@v2.10.1
       env:
         CIBW_ARCHS_MACOS: auto universal2
-        CIBW_SKIP: "cp311-* *-win32 pp38-* pp39-* *_i686 *-musllinux_*"
+        CIBW_SKIP: cp66-* pp37-win_amd64
```

Comment:
@ClementRomac I think this messed with the vtk again
cc @epaminondas

Reply:
from what I saw here  https://vtk.org/download/ :

- vtk doesn't seem to be available on win32, and since it is the default backend used for pyvista it makes it impossible to build a win32 version

- for linux, it is true that vtk is also not built against libpython 3.11 either (but you have to skip cp311 instead of limiting the scope of the compatible packages)

- on mac, it's the same, vtk not yet built for 3.11

=> maybe 
`CIBW_SKIP: cp311-* *-win32`
is OK

Reply:
Hmmm, yeah that didn't seem to solve everything (I just checked it out in the same branch again).

Reply:
I think the `MACOSX_DEPLOYMENT_TARGET: "10.13"` may be designed to stop this issue: 
```
        /Users/runner/work/simulate/simulate/third-party/fastwfc/src/run_wfc.cpp:97:41: error: 'value' is unavailable: introduced in macOS 10.13
              if (result.has_value() && (result.value().width > 0 || result.value().height > 0)) {
              ```

Reply:
I kept working on this in my other PR that I am merging. 🤞